docs/sources/mimir/configure/configure-resource-utilization-based-ingester-read-path-limiting.md

```diff
@@ -0,0 +1,28 @@
+---
+aliases:
+  - ../operators-guide/configure/configure-resource-utilization-based-ingester-read-path-limiting/
+description: Learn how to configure Grafana Mimir for resource utilization based ingester read path limiting.
+menuTitle: Resource utilization based ingester read path limiting
+title: Configure resource utilization based ingester read path limiting
+weight: 120
+---
+
+# Configure resource utilization based ingester read path limiting
```

Comment:
This doc page lacks some important information (e.g. metrics) which are documented in `docs/sources/mimir/references/architecture/components/ingester.md` instead. I think we should document the feature only in 1 place, either here or in `docs/sources/mimir/references/architecture/components/ingester.md`. Given it's not a critical feature (I mean something you absolutely need to be aware of) I would suggest to move all the doc here, and completely remove it from `docs/sources/mimir/references/architecture/components/ingester.md` (it's already listed among the list of production tips, which is good).

Reply:
Done.